docs/contents/components/forms/year-picker/index.ja.mdx

```diff
@@ -235,9 +278,16 @@ import "dayjs/locale/ja"
 
 ```tsx
 <VStack>
-  <YearPicker isReadOnly variant="outline" placeholder="outline" />
-  <YearPicker isReadOnly variant="filled" placeholder="filled" />
-  <YearPicker isReadOnly variant="flushed" placeholder="flushed" />
+  <For each={["outline", "filled", "flushed"]}>
```

Comment:
@kaehehehe 
It seems that using `<For />` has actually reduced readability. Why don't we try replacing `<For />` with something else?